app/controllers/company/tickets_controller.rb

```diff
@@ -25,14 +27,40 @@ def create
       end
     end
 
+    def resolution
+      @ticket = policy_scope([:company, Ticket]).find(params[:ticket_id]).decorate
+
+      if @ticket.complete!(ticket_resolution_params)
+        flash[:success] = 'Ticket resolved!'
+        redirect_to [:company, @ticket]
+      else
+        flash[:warning] = 'Ticket is resolved!'
+        render :show
+      end
+    end
+
+    def followed_new
+      @followed_ticket = policy_scope([:company, Ticket]).find(params[:ticket_id]).clone
+      @ticket = @followed_ticket.dup
+      @ticket.description = @followed_ticket.description
```

Comment:
```suggestion
      @ticket = Ticket.new(@followed_ticket.attributes.except(:something_not_needed))
```
А ще краще `@ticket = @original_ticket.follow_up`